src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -144,7 +147,7 @@ public GetAccessorSymbol(SynthesizedRecordEqualityContractProperty property)
 
             public override ImmutableHashSet<string> ReturnNotNullIfParameterNotNull => ImmutableHashSet<string>.Empty;
 
-            internal override bool HasSpecialName => _property.HasSpecialName;
+            internal override bool HasSpecialName => true;
```

Comment:
I don't understand this change, doesn't this correspond to the 'rtspecialname' metadata flag?

Reply:
`HasSpecialName` corresponds to the `specialname` metadata bit. Property accessors should set that bit.

---
In reply to: [447926980](https://github.com/dotnet/roslyn/pull/45336#discussion_r447926980) [](ancestors = 447926980)